### DISCUSSION TITLE
1. can I load video or image use model-view？

### DISCUSSION THREAD

**Discussion by kenchikuliu** [google/model-viewer#1908] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by kenchikuliu** [comment]
can video/Image show with model-view？

**Comment by elalish** [comment]
Can you be more specific? Generally `<img>` and `<video>` tags are used for displaying images and videos. `<model-viewer>` is for 3D models, though the results look a bit like images and videos, the difference being you can interact with them.

**Comment by AIRS-EXR** [comment]
> Can you be more specific? Generally `<img>` and `<video>` tags are used for displaying images and videos. `<model-viewer>` is for 3D models, though the results look a bit like images and videos, the difference being you can interact with them.

can I control the textrue of GLB model to achieve video play? Is model-viewer support?

**Comment by elalish** [comment]
No, glTF does not yet support video textures, so neither do we.

**Comment by kenchikuliu** [comment]
> No, glTF does not yet support video textures, so neither do we.

thx, how about add audio to model-viewer? can model-viewer support audio while glb model in ar mode?how can I achieve this?

**Comment by elalish** [comment]
Several model-viewer users have integrated audio; there's nothing special in our API for it. Simply play and pause your audio when you play and pause the animation. Of course, like with most AR customization, it will only work in webXR mode.

**Comment by 1kaiser** [comment]
@elalish how to take local glTF file for model viewer src. Visited the model editor legacy site but didn't understood how to.
As 
`<input type="file" id="input" multiple>`
Then there's
```
<model-viewer id="loading-demo" autoplay camera-controls interaction-prompt="none"
      src="../shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF-Binary/DamagedHelmet.glb" alt="A 3D model">
    </model-viewer>
```

How to say use src to local file with Dracodecoder support

**Comment by elalish** [comment]
Draco support is automatic. I'm not sure what you mean by local; the `src` URL above is a relative path, which means it's local to the website. If you mean local to your machine, then you need to run `npm run serve` from your website's directory so that your local files are served for testing.

**Comment by 1kaiser** [comment]
@elalish should it be ?
```
<input type="file" id="input" multiple>
<model-viewer id="loading-demo" autoplay camera-controls interaction-prompt="none"
      src="../shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF-Binary/DamagedHelmet.glb" alt="A 3D model">
    </model-viewer>
const viewer =   document.getElementById('loading-demo');
const fileURL = typeof file === 'string' ? file : URL.createObjectURL(file);
            viewer.src = fileURL

```
Here's when it opens in browser after loading all necessary scripts, then after input file is selected from local directory, the glb file doesn't loading as src to model viewer.
